subsys/bluetooth/controller/hci_driver.c

```diff
@@ -505,6 +505,33 @@ static int ble_enable(void)
 	return 0;
 }
 
+ISR_DIRECT_DECLARE(ble_controller_radio_isr_wrapper)
+{
+	ble_controller_RADIO_IRQHandler();
+
+	ISR_DIRECT_PM();
+	return 0; /* No need for rescheduling, because the interrupt handler
```

Comment:
Should probably put the block comment above.

Also coding standard says to do this:
```
/* multiline 
 * block comment
 */
```